Framework/NexusGeometry/src/NexusGeometrySave.cpp

```diff
@@ -277,12 +264,21 @@ inline void writeXYZPixeloffset(H5::Group &grp,
  * @param compInfo : componentInfo object.
  */
 inline void writeDetectorNumber(H5::Group &grp,
```

Comment:
inline specifier won't do anything here.

Reply:
I think this should somehow indicate that it is operating on a bank. Maybe call it `writeNXDetectorNumber`?